package-lock.json

```diff
@@ -12679,85 +12543,6 @@
       "funding": {
         "url": "https://github.com/sponsors/sindresorhus"
       }
-    },
-    "node_modules/yorkie": {
```

Comment:
Even though `yorkie` is a dependency of `@vue/cli-plugin-eslint` and not a top-level dependency, we also use it to ensure that bin/transifex/restructure.js is run:

https://github.com/getodk/central-frontend/blob/a54cf7bfd93c91cf6edd1126cddc2c7e365eac25/package.json#L16-L18

When it's installed, `yorkie` adds Git hooks locally. However, it doesn't seem to remove the hooks when it's uninstalled. When I try to make a commit after `yorkie` is uninstalled, I see:

```
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'node_modules/yorkie/src/runner.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.10.0

pre-commit hook failed (add --no-verify to bypass)
```

I'm surprised that you were able to make your later commits on this branch. Maybe you had switched to a different branch and reinstalled `yorkie`?

Interestingly, I do see that there's an uninstall script in the `yorkie` [`package.json`](https://github.com/yyx990803/yorkie/blob/a4cf01d789da2633a33a888b496fa35395e72109/package.json#L12). However, it [looks like](https://docs.npmjs.com/cli/v10/using-npm/scripts#a-note-on-a-lack-of-npm-uninstall-scripts) uninstall scripts are no longer run automatically.

In any case, since we want to continue using `yorkie` at least for now, I think we should install it as a top-level dependency.

Reply:
I've filed a related issue about possibly replacing `yorkie`: see #992.

Reply:
> I'm surprised that you were able to make your later commits on this branch. Maybe you had switched to a different branch and reinstalled `yorkie`?

It's likely I deleted the git hooks manually.